TestCases/hybrid_regression_AD.py

```diff
@@ -0,0 +1,238 @@
+    
+    ##############################################################
+    ### Unsteady Disc. adj. compressible RANS Windowed Average ###
+    ##############################################################
+
+    # Turbulent Cylinder
+    discadj_cylinder           = TestCase('unsteady_cylinder_windowed_average_AD')
+    discadj_cylinder.cfg_dir   = "disc_adj_rans/cylinder"
+    discadj_cylinder.cfg_file  = "cylinder_Windowing_AD.cfg" 
+    discadj_cylinder.test_iter = 9
+    discadj_cylinder.test_vals = [3.004402]
+    discadj_cylinder.unsteady  = True
+    test_list.append(discadj_cylinder)
+    
+    ##########################################################################
+    ### Unsteady Disc. adj. compressible RANS DualTimeStepping 1st order   ###
+    ##########################################################################
+
+    # Turbulent Cylinder
+    discadj_DT_1ST_cylinder           = TestCase('unsteady_cylinder_DT_1ST')
+    discadj_DT_1ST_cylinder.cfg_dir   = "disc_adj_rans/cylinder_DT_1ST"
+    discadj_DT_1ST_cylinder.cfg_file  = "cylinder.cfg"
+    discadj_DT_1ST_cylinder.test_iter = 9
+    discadj_DT_1ST_cylinder.test_vals = [3.698167, -1.607051, -0.002159, 0.000028]
+    discadj_DT_1ST_cylinder.unsteady  = True
+    test_list.append(discadj_DT_1ST_cylinder)
+
+    ######################################################
+    ### Unsteady Disc. adj. compressible pitching NACA ###
+    ######################################################
+
+    # compressible pitching NACA0012
+    discadj_pitchingNACA0012           = TestCase('pitchingNACA0012')
+    discadj_pitchingNACA0012.cfg_dir   = "disc_adj_euler/naca0012_pitching"
+    discadj_pitchingNACA0012.cfg_file  = "inv_NACA0012_pitching.cfg"
+    discadj_pitchingNACA0012.test_iter = 4
+    discadj_pitchingNACA0012.test_vals = [-1.219713, -1.645717, -0.007513, 0.000013]
+    discadj_pitchingNACA0012.unsteady  = True
+    test_list.append(discadj_pitchingNACA0012)
+
+    #######################################################
+    ### Disc. adj. turbomachinery                       ###
+    #######################################################
+    
+    # Transonic Stator 2D
+    discadj_trans_stator           = TestCase('transonic_stator')
+    discadj_trans_stator.cfg_dir   = "disc_adj_turbomachinery/transonic_stator_2D"
+    discadj_trans_stator.cfg_file  = "transonic_stator.cfg" 
+    discadj_trans_stator.test_iter = 79
+    discadj_trans_stator.test_vals = [79.000000, -1.938806, -1.995540]
+    test_list.append(discadj_trans_stator)
+    
+    ###################################
+    ### Structural Adjoint          ###
+    ###################################
+   
+    # Structural model
+    discadj_fea           = TestCase('discadj_fea')
+    discadj_fea.cfg_dir   = "disc_adj_fea"
+    discadj_fea.cfg_file  = "configAD_fem.cfg" 
+    discadj_fea.test_iter = 4
+    discadj_fea.test_vals = [2.183540, 2.071459, -0.000363, -8.655000]
```

Comment:
```suggestion
    discadj_fea.test_vals = [1.774569, 1.928023, -0.000364, -8.690300]
```
Seems like these values are repeatable after preaccumulation was enabled.

Reply:
I have not observed any variations either